dts/bindings/i2s/intel,ssp-settings.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Intel SSP generic settings.
+
+compatible: "intel,ssp-settings"
+
+include: base.yaml
+
+properties:
+    force-dynamic-clock-gating:
+      type: boolean
+      description: If the platform needs to force dynamic clock gating.
+
+    ports-base:
+      type: int
+      description: Number of base SSP ports available.
+
+    ports-extended:
+      type: int
+      description: Number of extended SSP ports available.
```

Comment:
It is not a blocker for me, but I really find weird dt using for settings. DT should describe HW and general settings should come from Kconfig IMHO.

Reply:
Well, isn't the number of ports a property of the HW? Before this patch, it was being decided on HW version alone...